code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -124,6 +124,47 @@
 	stamina = 13 //Total of 78 with less falloff (very big)
 	sharpness = SHARP_NONE
 
+// Normal airburst
+
+/obj/item/projectile/bullet/pellet/shotgun_airburst_casing
+	name = "airburst rubber casing"
+	damage = 2 // The shell for the bullets, not designed to hit people
+	range = 7 // roughly 4 tiles 
+	sharpness = SHARP_NONE
+
+/obj/item/projectile/bullet/pellet/shotgun_airburst_casing/on_range()
+	var/obj/item/ammo_casing/shotgun/airburst_pellet_shell/P = new(get_turf(src))
+	var/mob/living/L = new (get_turf(src))//it's jank, but casings can only be shot via a mob's location
+	P.fire_casing(get_edge_target_turf(firer, get_dir(firer, original)), L)
```

Comment:
Even if it isn't used currently, ``fired_from`` can and should be the arg used for admin logs to tell who shot something. It's currently done for chemical darts, but I think these bullets lack critical admin logging.